apps/web/src/constants/talentSearchConstants.ts

```diff
@@ -1,5 +1,5 @@
 export const TALENTSEARCH_SUPPORT_EMAIL =
   (process.env.TALENTSEARCH_SUPPORT_EMAIL as string) ??
-  "gctalent-talentgc@support-soutien.gc.ca";
+  "support-soutien@talent.canada.ca";
```

Comment:
Less a request and more a question, why is this like it is?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5f673fd3-5caa-4092-bd91-a56bd2da29ca)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5ddd0ad1-7899-4a1a-80db-c367b3432757)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/1ac85246-df33-44a7-96a9-d442afb66e10)

There is no environment variable `TALENTSEARCH_SUPPORT_EMAIL` so all it does it return the fallback value in which case why refer to the environment variable in the first place rather than just return a constant from the start?

Reply:
Environment variables

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/b5c5fec9-8b89-464d-bc3c-2fe43693c643)

Reply:
> Less a request and more a question, why is this like it is?
> 
> There is no environment variable `TALENTSEARCH_SUPPORT_EMAIL` so all it does it return the fallback value in which case why refer to the environment variable in the first place rather than just return a constant from the start?

The reason for the environment variable is so that different values can be uses in different environment. For example, if a user wanted to test having a different email address (ex., support-soutien-test@talent.canada.ca) to send to without having to alter the code on the UAT, we would be able to switch the value without committing code.